packages/rest/src/__tests__/integration/rest.server.integration.ts

```diff
@@ -646,7 +646,6 @@ paths:
         cert: fs.readFileSync(certPath),
       },
     });
-    server.handler(dummyRequestHandler);
```

Comment:
This looks suspicious to me. Could you please explain the reasoning for this change?

Reply:
this line removed cause the previous PR introduced SwaggerRedirect , 
removing it make this line come back again